src/test/ui/parser/macro/issue-33569.rs

```diff
@@ -1,3 +1,5 @@
+fn main() {}
```

Comment:
Nit: I personally always write `main` in the end (it's usually not the most important part of the test), so this looks weird to me.

Reply:
I usually put it first in case I need to add more to the test so it disturbs .stderr less basically.

Reply:
How can `main` in the beginning or in the end disturb `.stderr`?

Reply:
Hmm; I didn't think it through; you're right, it can't.